### DISCUSSION TITLE
Get files from ISO

### DISCUSSION THREAD

**Discussion by Shlepsss** [CosmosOS/Cosmos#2847] [Q&A]
When im trying to get files from ISO with 1:/ this not working. There's no files in 1:/ How to get this files with OS

**Comment by retrievaaaa** [comment]
Are you on latest devkit?

**Reply by Shlepsss** [reply]
yes

**Reply by retrievaaaa** [reply]
So you are on 3cddfe759b4f2a88ab0b0000f5a40c93655a76ad and its not working? Weird. Can you post the .iso file that Cosmos generates here?

**Reply by Shlepsss** [reply]
[qeeweOS.zip](https://github.com/CosmosOS/Cosmos/files/13663911/qeeweOS.zip)
Github deny ISO files, so i put this into .zip

**Reply by retrievaaaa** [reply]
Looking at this file you are definitely not on the latest devkit, make sure to update. The ISO9660 file system was just recently (1-2 days ago) fixed.

**Comment by Shlepsss** [comment]
> Looking at this file you are definitely not on the latest devkit, make sure to update. The ISO9660 file system was just recently (1-2 days ago) fixed.

DUDE! YOU BEST!!! IT WORKED!!! THANK YOU VERY MUCH!! i have installed the new version of cosmos!!
